package.json

```diff
@@ -52,6 +52,7 @@
     "mocha": "^2.2.5",
     "mocha-jsdom": "^1.0.0",
     "react": "^0.14.0",
+    "prop-types": "^15.5.0",
```

Comment:
1. I think, there is no need to put it to dependencies if react is not there.
2. Nice catch! Thanks :)
3. Not sure if I understand what you mean by that.